storage/replica.go

```diff
@@ -1011,6 +1015,7 @@ func (r *Replica) applyRaftCommand(ctx context.Context, index uint64, originRepl
 		// time it's required.
 		if _, ok := ba.GetArg(roachpb.TruncateLog); ok {
 			r.setCachedTruncatedState(nil)
+			r.raftEntryCache.clearAll()
```

Comment:
Yes, but why do it in two different places? Shouldn't one or the other be enough?